Pipfile

```diff
@@ -25,8 +21,12 @@ requests = "==2.24.0"
 requests-oauthlib = "==1.3.0"
 sqlparse = "==0.3.1"
 urllib3 = "==1.25.10"
+<<<<<<< HEAD
```

Comment:
Same here - merge issues? (lines 26 and 29 below, too?)

Reply:
Removed all of the merge conflicts.